example/set.wren

```diff
@@ -0,0 +1,110 @@
+class Set {
+  new {
+    _list = []
+    _clean = true
+  }
+
+  new (list) {
```

Comment:
Style nit, but I think style nits matter for examples. Can you remove the space before `(`?